src/View/CategoryItem/add.html.twig

```diff
@@ -0,0 +1,31 @@
+
+	<div class="container">
+		<h1 class="text-center">Ajoutes une nouvelle catégorie</h1>
```

Comment:
increment h tags + 1 since you can ony have one h1 by page (the one in categoryItems index)